src/mono/mono/mini/mini.h

```diff
@@ -2886,6 +2886,18 @@ typedef enum {
 	SIMD_OP_LLVM_I16ABS,
 	SIMD_OP_LLVM_I32ABS,
 	SIMD_OP_LLVM_I64ABS,
+	SIMD_OP_LLVM_I8ABS_SATURATE,
+	SIMD_OP_LLVM_I16ABS_SATURATE,
+	SIMD_OP_LLVM_I32ABS_SATURATE,
+	SIMD_OP_LLVM_I64ABS_SATURATE,
+	SIMD_OP_LLVM_I4FABSOLUTE_COMPARE_GREATER_THAN,
```

Comment:
should it be `SIMD_OP_LLVM_FABSOLUTE_COMPARE_GREATER_THAN` (without `I4` in it)?

Reply:
Opps, yeah.